docs/core_docs/docs/integrations/vectorstores/upstash.mdx

```diff
@@ -0,0 +1,45 @@
+import CodeBlock from "@theme/CodeBlock";
+import CreateClientExample from "@examples/indexes/vector_stores/upstash/create_client.ts";
+import IndexQueryExample from "@examples/indexes/vector_stores/upstash/index_and_query_docs.ts";
+import DeleteExample from "@examples/indexes/vector_stores/upstash/delete_docs.ts";
+
+# Upstash Vector
+
+Upstash Vector is a REST based serverless vector database, designed for working with vector embeddings.
+
+## Setup
+
+1. Create Upstash Vector Index
+
+You can create an index from [Upstash Console](https://console.upstash.com/vector). For further reference, see [docs](https://upstash.com/docs/vector/overall/getstarted).
+
+2. Install Upstash Vector SDK.
+
+```bash npm2yarn
+npm install -S @upstash/vector
+```
+
+We use OpenAI for the embeddings of the below examples.
+However, you can also create the embeddings using the model of your choice, that is available in the LangChain.
+
+```bash npm2yarn
+npm install -S @langchain/openai
+```
```

Comment:
Use this tooltip & also show to install `@langchain/community`
(remove back slash when inserting into docs)

```mdx
import IntegrationInstallTooltip from "@mdx_components/integration_install_tooltip.mdx";

<IntegrationInstallTooltip></IntegrationInstallTooltip>

\`\`\`bash npm2yarn
npm install @langchain/openai @langchain/community
\`\`\`
```